### ISSUE TITLE
dl fail

### ISSUE THREAD

**Issue by JacenHorn** [mmallett/pushups#3]
Good afternoon,
I'm really interested in using your sftwr on my new FitBit. 
Unfortunately, I cannot get it to install; having done syncs, refreshes, restarts, etc.
Any suggestions?  TY 
-Jacen
Versa 2, ver 35.70.7.14, 1.1GB available

**Comment by mmallett** [comment]
Interesting. A few questions.

Is it only the Push-ups app you're having trouble installing?

Have you been able to successfully install the Push-ups app before?

Have you tried the generic Fitbit troubleshooting steps?

- Restarting the watch
- Restarting Bluetooth on your phone

Fitbit has collected a [list of steps for general app troubleshooting here](https://help.fitbit.com/articles/en_US/Help_article/2275.htm)

I see there is a [Fitbit forum post where one user experienced a similar issue](https://community.fitbit.com/t5/Versa-Smartwatches/Can-t-install-apps-or-clock-face/td-p/2643703). Their resolution was to delete the app before trying to re-download it.

**Comment by JacenHorn** [comment]
Good day!

Yes, indeed, only this app.

My watch syncs and I have 20 other apps from 3rd party Devs.  All with
expected levels of success.

I consider myself tech savvy.  I searched through normal Fitbit TS first,
conducting a watch and phone restart (after resetting settings, like your
suggestions below).

I only just discovered your App today.  I tried uninstalling a different
app to see if the Fitbit store was experiencing problems.  That worked.
The only other hiccups I've noticed have been sync issues if my phone is
not on the same Wi-Fi as the watch or longer than expected install times;
but this was left for greater than 20min a few different times.

Sometimes during a phone app restart (closing it or even attempting a Force
Stop) when I enter the My Apps tab I get a pop-up asking if I'd like to
continue Install (as it perpetually never finishes -- thereby not providing
a 'check mark' next to it in the All Apps tab).

I appreciate you taking the time to respond.

I will likely try another full shutdown this evening and attempt to sync
while charging (battery is currently above 50 on both devices).  I am
running on a fully updated Note 8.

Thank you,
Jacen
